numba/misc/numba_sysinfo.py

```diff
@@ -585,7 +590,11 @@ class DisplaySeqMaps(tuple):
         ("__Python Information__",),
         DisplayMap({k: v for k, v in info.items() if k.startswith('Python')}),
         ("",),
+        ("__Numba Information__",),
+        ("Numba Version", info.get(_numba_version, '?')),
+        ("",),
         ("__LLVM Information__",),
+        ("llvmlite Version", info.get(_llvmlite_version, '?')),
```

Comment:
Am wondering if Numba, llvmlite and LLVM information ought to go under a single header of e.g. `__Numba Toolchain Versions__` or something? I'm not sure  that `llvmlite` belongs under `LLVM` as is?

Reply:
Yeah, that would be better.